site/slack.go

```diff
@@ -41,6 +51,10 @@ func (bot SlackBot) testMessage() {
 }
 
 func (bot SlackBot) sendKnock(username string, location string) (messagets string) {
+	if !bot.isValidBot {
+		log.Printf("SlackBot not valid; ignoring request for sendKnock.")
+		return
+	}
```

Comment:
I would check how noisy this log is, I think it's acceptable to log only when the bad configuration is detected at startup, and then just silently return, so that's an option if this pollutes logs